target/linux/lantiq/xrx200/base-files/lib/preinit/05_set_preinit_iface_lantiq

```diff
@@ -0,0 +1,15 @@
+set_preinit_iface() {
+	ip link set eth0 up
+
+	case $(board_name) in
+	avm,fritz7412|\
+	netgear,dm200)
+		ifname=lan
```

Comment:
This is not needed any more with:
https://patchwork.ozlabs.org/project/openwrt/list/?series=249780